api/cloudbuild.yaml

```diff
@@ -56,6 +56,7 @@ steps:
       - DB_PASS=$_DB_PASS
       - DB_PORT=$_DB_PORT
       - DB_USER=$_DB_NAME
+      - BASE32_SECRET=$_BASE32_SECRET
       - AZURE_DB_HOST=$_AZURE_DB_HOST
```

Comment:
Just remove these azure things, so they don't get added back in

Reply:
Good catch!